packages/itinerary-body/src/__mocks__/itineraries/otp2-with-fareproducts.json

```diff
@@ -728,7 +728,6 @@
         "length": 368
       },
       "steps": [],
-      "routeShortName": "101",
```

Comment:
Was this removed for testing?

Reply:
Yes! The story now shows how we handle a route with no short name